src/NBitcoin/RPC/IRPCClient.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NBitcoin.RPC
+{
+    /// <summary>
+    /// Interface for an RPC client.
+    /// </summary>
+    /// <remarks>Extend where necessary.</remarks>
+    public interface IRPCClient
+    {
+        /// <summary>Send a command over RPC.</summary>
+        /// <param name="commandName">The command to execute.</param>
+        /// <param name="parameters">An array of command parameters.</param>
+        /// <remarks>See https://en.bitcoin.it/wiki/Original_Bitcoin_client/API_calls_list </remarks>
```

Comment:
Use `<see cref="https://en.bitcoin.it/wiki/Original_Bitcoin_client/API_calls_list"/>` instead.

same on other lines in this file. 

Though I think it will be better to put that link in the interface remarks instead of adding it to every single method's description.

Reply:
I will do that instead.